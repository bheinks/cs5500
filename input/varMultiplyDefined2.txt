(*
	Variable is multiply defined in same declaration line
*)

program p;

var x,y,x,z : integer;

begin
  write(5)
end.

