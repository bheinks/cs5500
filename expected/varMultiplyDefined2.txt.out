

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding x to symbol table with type INTEGER

+++ Adding y to symbol table with type INTEGER

+++ Adding x to symbol table with type INTEGER
Line 7: Multiply defined identifier
